src/tekhsi/tek_hsi_connect.py

```diff
@@ -489,11 +489,25 @@ def done_with_data(self) -> None:
         self._done_with_data_release_lock()
 
     def force_sequence(self) -> None:
-        """force_sequence asks the instrument to please give us access.
-
-        to the current acquisition data. This is useful when connecting to a stopped instrument to
+        """force_sequence asks the instrument to please give us access to the already acquired data. 
+        This is useful when connecting to a stopped instrument to
         get access to the currently available data. Otherwise, the API will wait until the next
         acquisition.
+
+        Examples:
+            from tm_data_types import AnalogWaveform, write_file
```

Comment:
Each line of this should start with the following characters: `>>> `
```suggestion
            >>> from tm_data_types import AnalogWaveform, write_file
```